nilearn/image/resampling.py

```diff
@@ -340,7 +340,7 @@ def resample_img(
     order="F",
     clip=True,
     fill_value=0,
-    force_resample=False,
+    force_resample=True,
```

Comment:
Well this is an API change: it has to go through a deprectaion cycle befire we can actually  publish this.